lumigator/schemas/lumigator_schemas/jobs.py

```diff
@@ -193,6 +193,7 @@ class JobResponse(BaseModel, from_attributes=True):
     name: str
     description: str
     status: JobStatus
+    job_type: JobType
```

Comment:
❓ could you help me understand why we need to add `job_type`?

Reply:
This info is present in the database record, and I used it for job traversal. Even though I don't use it anymore, there is no need to omit it from the job record, and it will most probably be useful later on to return it with the rest of the job information.